### PR TITLE
add support log guide

### PR DESCRIPTION
**Pull request by northyg**
WIP but the concepts are solid. Added TODO in places where could add more links and/or details. Appreciate comments, additions or corrections. :)

### CODE REVIEW

File: content/departments/support/tools/support-logging-guide.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+# Application Engineers guide to getting started with Sourcegraph logging, services, and observability
+
+**Note to reader:** This guide assumes little to no prior knowledge of how to troubleshoot distributed systems, gather logs, levels of user access, etc. It is an overview with the goal to provide a starting point for _where to look, why to look there, what you are looking for, and who can provide it._
+
+### At the end of this guide you will...
+
+- Have gained a conceptual understanding of Sourcegraph architecture, services, and the logs they generate
+- Learn how to grep through logs and find what you are looking for
```
Comment (Kelvin-Lee) [inline content/departments/support/tools/support-logging-guide.md]: maybe put grep in the ``grep`` backticks, and maybe say something like:

"Learn how to `grep`/search/parse through logs"

To give them an idea of what it means?

File: content/departments/support/tools/support-logging-guide.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+- Know the Sourcegraph logging levels and how to set them
+- Be better equiped to understand when to gather logs and how to interpret them
```
Comment (Kelvin-Lee) [inline content/departments/support/tools/support-logging-guide.md]: equiped -> equipped (one extra 'p')

File: content/departments/support/tools/support-logging-guide.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+- Become familiar with other Sourcegraph observabiltiy tools
+
+### Sourcegraph architecture
+
+Sourcegraph is essentially built from many different **services** that are deployed from **docker images**. Each **service** has its own resource allocation, configuration, and other **services** it talks to. Some services communicate to **external services** like Github through API's, while others communicate to **internal services**, such as a database, or the client aka **browser**. Each service generates its own **logging data.** All services collect at the **frontend** service. See the Sourcegraph Architecture page for a visual of each service. TODO: Add link to arch page
```
Comment (Kelvin-Lee) [inline content/departments/support/tools/support-logging-guide.md]: Addressing your TODO: For your convenience: https://docs.sourcegraph.com/dev/background-information/architecture

File: content/departments/support/tools/support-logging-guide.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+We can extrapolate that the user performed an action in Sourcegraph that used CodeIntel (hover tool tips?) and the request completed, but was slow and user probably saw it 'take awhile to load the tooltip.' This _could_ indicate Sourcegraph is under-provisioned and could use more resources. Or it could indicate a particular user is having issues. Check for clues like are the slow requests being sent to one user or many? Note: the `user` value can be looked up in the pgsql database table!
+
+TODO: Add more log examples and how to interpret?
+
+TODO: Need another doc, etc pointing to how to provision Sourcegraph more better and provide guidance to site-administrators, considering deployment types, repo type, etc
+
+### Further reading
+
+TODO: ADD anymore cool links
```
Comment (Kelvin-Lee) [inline content/departments/support/tools/support-logging-guide.md]: 😎

File: content/departments/support/tools/support-logging-guide.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+- Is the question more of a "how do I..." type question, such as "This query ... is not returning any results, what am I doing wrong?" Or "my batch spec is failing with this error message..." Consider asking for more details on what they are attempting and testing it before asking for logs.
+
+##### Accessing logs
+
+The way to access logs depends on the deployment method and which service you need logs for. For quick reference see the [Support command generator tool](https://sourcegraph.github.io/support-generator/)
+
+TODO: Add more specifics on commands to gather logs by deployment type, etc
+
+[Logging levels are set as an environmental variable `SRC_LOG_LEVEL`.](https://docs.sourcegraph.com/admin/observability/logs) The default logging level in cluster deployments is `dbug`, and in docker it is `warn`. **Site administrators** may have changed the logging level to _more_ or _less_ verbose. The most verbose logging level is `dbug` and the least is `crit`.
```
Comment (northyg) [inline content/departments/support/tools/support-logging-guide.md]: TODO: HOW to set the log level

File: content/departments/support/tools/support-logging-guide.md (hunk 1 above)
Comment (northyg) [reply content/departments/support/tools/support-logging-guide.md]: Done!

File: content/departments/support/tools/support-logging-guide.md (hunk 2 above)
Comment (northyg) [reply content/departments/support/tools/support-logging-guide.md]: Good call! Now that I'm think about it, I should throw everything in a google doc for typos 😅

File: content/departments/support/tools/support-logging-guide.md (hunk 5 above)
Comment (northyg) [reply content/departments/support/tools/support-logging-guide.md]: I linked this doc https://docs.sourcegraph.com/admin/install/docker-compose/operations#set-environment-variables

**Pull request merged**